src/wrap.lua

```diff
@@ -73,8 +73,13 @@ modutil.mod.Path.Wrap("IsRarityForcedCommon", function(base, name, args)
 end)
 
 modutil.mod.Path.Wrap("GetRarityChances", function(base, loot)
-    if config.HadesRarity == false then return DefaultHadesRarity end
+    --Adding this as another check, not *really* needed but maybe keep it as a safeguard?
+    --So that if rarity = 0 do nothing
+    if config.MinimumRarity == 0 then return end
```

Comment:
We wouldn't actually want this here, because this would prevent all the rarity overrides above it from happening. An example would be if you still want commons, but want everything above it to be a 50% chance (weird but valid)

This would make sense with a setting called MaximumRarity, though